bindata/assets/alerts/kube-apiserver-slos-basic.yaml

```diff
@@ -49,23 +49,23 @@ spec:
           (
             sum(rate(apiserver_request_duration_seconds_count{job="apiserver",verb=~"LIST|GET",subresource!~"proxy|log|exec",scope="resource"}[5m]))
           -
-            (sum(rate(apiserver_request_duration_seconds_bucket{job="apiserver",verb=~"LIST|GET",subresource!~"proxy|log|exec",scope="resource",le="0.1"}[5m])) or vector(0))
+            (sum(rate(apiserver_request_duration_seconds_bucket{job="apiserver",verb=~"LIST|GET",subresource!~"proxy|log|exec",scope="resource",le="1"}[5m])) or vector(0))
```

Comment:
if scope="resource", then verb should always be `GET`, and if scope is `namespace` verb should always be `LIST`, no? 

when we sync the alert with upstream then we can fix these. for now, let's merge it as is

Reply:
that's correct and yeah that might be worth fixing upstream although they are not really differentiating between verb but rather "read" and "write" requests. So they are just taking "LIST|GET" as the verbs that would refer to read requests no matter the scope